seccomp.json

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
I forgot to mention, this is of course our projectatomic/docker default (so we can run systemd as well)